### DISCUSSION TITLE
可以参考使用我的 amll-lyric 模块实现更多歌词格式的加载

### DISCUSSION THREAD

**Discussion by Steve-xmh** [waylyrics/waylyrics#232] [Ideas]
https://crates.io/crates/amll-lyric
同时咱们还有大量高质量的逐词歌词数据库：https://github.com/Steve-xmh/amll-ttml-db
（即便现在不支持逐词也没关系，可以自行合并单词变成普通的逐行歌词）

**Comment by mokurin000** [comment]
看起来很不错，不过feeluown现在除了QQ音乐/网易云/YT Music，其他来源支持都停摆了（目前Linux支持中文用户主流在线音乐平台最多的播放器）
等之后Waylyrics支持逐词的话会考虑接入db获取逐词歌词的，现在获取的id会直接走 ncmapi-rs/QQMusicApi 获取歌词

**Reply by mokurin000** [reply]
> 之后Waylyrics支持逐词

不过可能会是重新开的项目了，Waylyrics现在使用GTK最大的原因是 GdkSurface 对Wayland下穿透设置控制比较方便
或者把现在label的逻辑改成 canvas，另外的前置条件是 #231 ，因为时间抖动对于逐词歌词影响会比较大.... 或者就是每一句不论如何都不影响播放动画，之后再设置新的句子
